parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -560,6 +560,13 @@ configGPUDrivers() {
     if [[ $OS == $UBUNTU_OS_NAME ]]; then
         retrycmd_if_failure 120 5 25 nvidia-modprobe -u -c0 || exit $ERR_GPU_DRIVERS_START_FAIL
     fi
+
+    # Noticed that NVIDIA driver version R515 and later versions will experience long loading time of nvidia-smi on 
+    # ND A100-v4 and NDm A100-v4 series. For now, use the workaround solution to enable NVIDIA persistence mode which 
+    # will accelerate the loading time of future nvidia-smi operations
+    if [[ $OS == $MARINER_OS_NAME ]]; then
+        retrycmd_if_failure 120 5 35 nvidia-persistenced --persistence-mode || exit $ERR_GPU_DRIVERS_START_FAIL
```

Comment:
is this sufficient for reboots, cc @henryli001 ?

should we install a systemd service and enable + start that instead?

Reply:
Reboots will reset/turn off persistence. Question is do we want to enable persistenced by default for all SKUs, and always

Reply:
Can create a systemd service for this, if we want to have above experience

Reply:
+1 to what Ganeshkumar's response. I don't have customer usage data to justify whether this should be generalized back when I developed this, so I only implemented in this way so as to resolve the provisioning error.